backend/src/app/tasks/file_gc.clj

```diff
@@ -45,7 +46,8 @@
   (fn [params]
     (db/with-atomic [conn pool]
       (let [min-age (or (:min-age params) (:min-age cfg))
-            cfg     (assoc cfg :min-age min-age :conn conn)]
+            id      (:id params)
```

Comment:
can this id be destructured directly in params?

Reply:
done